packages/e2e-tests/specs/editor/various/copy-cut-paste-whole-blocks.test.js

```diff
@@ -100,6 +100,7 @@ describe( 'Copy/cut/paste of whole blocks', () => {
 		await page.click( '.editor-block-list-item-paragraph' );
 		await page.keyboard.type( 'P' );
 		await page.keyboard.press( 'ArrowLeft' );
+		await page.evaluate( () => new Promise( requestAnimationFrame ) );
```

Comment:
Is this is a Puppeteer smell, or a "us" smell? 😄

Reply:
This is us. We are doing some delays with `requestAnimationFrame`, so I'll comment

Reply:
I wonder if this would stop being an issue after the switch to Playwright?

Reply:
Not sure. I don't mind adding the delay in tests if we are adding the delay.